edx/edx-app-ios
# Use font palette in OEXStyles

### Description

Use OEXStyles' font palette. 
### Notes

OEXStyles.m contains the app-wide font palette. This PR replaces `UIFont.*ontWithName` methods with the equivalent existing `OEXStyles.sharedStyles.*ontWithName` methods. There are NO changes to the UI, this is purely shifting the code to rely on OEXStyles as the programmatic source of the font palette. 
### How to test this PR

Run the test suite to verify that none of the screenshots have changed. This will confirm that the UI has not changed.
### Reviewers

If you've been tagged for review, please "Start a review" with the Github GUI.
